ckanext/unaids/dataset_transfer/logic.py

```diff
@@ -60,16 +60,16 @@ def send_dataset_transfer_emails(dataset_id, recipient_org_id):
             DatasetTransferRequest.dataset_id == dataset_id,
             DatasetTransferRequest.recipient_org_id == recipient_org_id
         ).all()
-    users_to_email = _get_users_to_email(
-        recipient_org=recipient_org,
+    users_to_email = get_org_admins_with_email_addresses(
+        org=recipient_org,
         exclude_user_ids=user_ids_already_emailed
     )
 
     for user in users_to_email:
         try:
-            subject = 'testing123'
+            subject = _('Dataset Transfer')
             body = render_jinja2(
-                'email/dataset_transfer.html',
+                'email/dataset_transfer.txt',
                 extra_vars={
                     'user': user,
                     'dataset': 'dataset',
```

Comment:
```
 'dataset': 'dataset',
```
How is this supposed to work? Jinja template is referencing `dataset.name` so the whole e-mail request is probably going to throw an template exception.

Reply:
No no. If you send the `dataset` object to jinja2, then in jinja2 you can access the object like `dataset.name` without any issues.